src/components/DropdownMenu/DropdownMenu.js

```diff
@@ -79,7 +79,14 @@ export default class DropdownMenu extends React.Component {
                 this.setState({ active: false });
                 this.props[`select${i}`]();
               }}>
+              {this.props[`iconLeft${i}`]
+                && <i className={`${DROPDOWN_CLASSES.ICON}${this.props[`iconLeft${i}`]} -ml--0 -mr--1 `} aria-hidden="true"></i>
+              }
               {itemContent}
+              {this.props[`iconRight${i}`]
+                // eslint-disable-next-line max-len
+                && <i className={`chi-icon icon-${this.props[`iconRight${i}`] === DROPDOWN_CLASSES.ICON_CHEVRON_RIGHT ? `${this.props[`iconRight${i}`]} -ml--auto` : `${this.props[`iconRight${i}`]} -ml--2`} -mr--0`} aria-hidden="true"></i>
```

Comment:
We can make it more scalable, readable and avoid disabling the linter by simply creating variables with the value of the icons:

![image](https://user-images.githubusercontent.com/86958774/180975538-bb4f8824-902a-4b25-9b12-bd779296a661.png)
![image](https://user-images.githubusercontent.com/86958774/180976409-b6fac216-e276-44da-af78-98ea82521eaf.png)

```suggestion
              {iconRight}
```